spec/controllers/request_controller_spec.rb

```diff
@@ -1048,6 +1048,18 @@ def expect_hidden(hidden_template)
     expect(assigns[:outgoing_message].body).to include(default_message.strip)
   end
 
+  it 'allows the default text to be set via the default_letter param' do
+    get :new, :public_body_id => @body.id, :default_letter => "test"
```

Comment:
We use `url_name` in the [documentation](http://alaveteli.org/docs/developers/api/#starting-new-requests-programmatically), so I think this spec should reflect that.